### PR TITLE
clean up motd template

### PR DESCRIPTION
**Pull request by Pandademic**
make it look nicer.

fluff , basically

### CODE REVIEW

**Review by mszoek — COMMENTED** [review]
This just seems like extra words to me. Does it not have the same meaning as it did before?

**Comment by Pandademic** [comment]
@mszoek not really , i think it just looks a little bit nicer.

File: etc/motd.template [hunk 1]
```diff
@@ -2,8 +2,9 @@ Welcome to ravynOS __VERSION__ (__CODENAME__)!
 
 ** DEVELOPER PREVIEW RELEASE **
 
-Although airyxOS is mostly compatible with FreeBSD, please
+
+Although ravynOS is mostly compatible with FreeBSD, please
 do not report issues to the FreeBSD developers or bother the FreeBSD developers!
```
Comment (mszoek) [inline etc/motd.template:7]: OK - how about we change this to:
`do not bother the FreeBSD developers with ravynOS issue reports!`

**Review by mszoek — CHANGES_REQUESTED** [review]

File: etc/motd.template (hunk 1 above)
Comment (Pandademic) [reply etc/motd.template:7]: yes , that makes senese

**Comment by Pandademic** [comment]
how's this now?

**Review by mszoek — APPROVED** [review]

**Comment by mszoek** [comment]
LGTM!

**Pull request merged**